recipes/optuna-dashboard/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "optuna-dashboard" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optuna-dashboard-{{ version }}.tar.gz
+  sha256: 277a81b0ebb33f405dfba2cd15d2ec628abd7b4acacd6364c88cea04ee3fa907
+
+build:
+  number: 0
+  entry_points:
+    - optuna-dashboard = optuna_dashboard.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - bottle
+    - optuna >=2.7
+    - python
+    - typing-extensions  # [py<38]
+
+test:
+  imports:
+    - optuna_dashboard
+    - tests
+  commands:
+    - pip check
+    - optuna-dashboard --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/optuna/optuna-dashboard
+  summary: Real-time dashboard for Optuna.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - c-bata
+    - g-votte
+    - HideakiImamura
+    - hvy
+    - iwiwi
+    - not522
+    - sile
+    - smly
+    - toshihikoyanase
+    - ytsmiling
```

Comment:
@c-bata @g-votte @HideakiImamura @hvy @iwiwi @not522 @sile @smly @toshihikoyanase @ytsmiling, please let us know here if you are ok being added as maintainers on this feedstock. Thanks! 😄

Reply:
Hi @jakirkham! Thank you for your confirmation. All recipe-maintainers now put a reaction :smile: